userguide/one_shot_object_detection/README.md

```diff
@@ -0,0 +1,109 @@
+# One-Shot Object Detection
+
+One-Shot object detection is the task of learning to detect a set of categories from a handful of images. Unlike the Object Detector which requires many varied examples of objects in the real world, the One-Shot detector only requires a single canonical example of an object to train the classifier on and performs predictions for that category in the wild.
```

Comment:
I would probably say something like "from as few as one example per category" rather than "a set of categories from a handful of images"

Reply:
That sounds better, thanks!